python/cudf_polars/tests/expressions/test_stringfunction.py

```diff
@@ -150,3 +150,40 @@ def test_slice_column(slice_column_data):
     else:
         query = slice_column_data.select(pl.col("a").str.slice(pl.col("start")))
     assert_ir_translation_raises(query, NotImplementedError)
+
+
+def test_replace_literal(ldf):
+    query = ldf.select(pl.col("a").str.replace("A", "a", literal=True))
+    assert_gpu_result_equal(query)
+
+
+def test_replace_re(ldf):
+    query = ldf.select(pl.col("a").str.replace("A", "a", literal=False))
+    assert_ir_translation_raises(query, NotImplementedError)
+
+
+@pytest.mark.parametrize(
+    "target,repl",
+    [
+        (["A", "de", "kLm", "awef"], "a"),
+        (["A", "de", "kLm", "awef"], ["a", "b", "c", "d"]),
+        # Test an expression, cannot use pl.col('a') since it contains nulls
+        # which replace_many doesn't support
+        (
+            pl.lit(pl.Series(["A", "de", "kLm", "awef"])),
+            pl.lit(pl.Series(["a", "b", "c", "d"])),
+        ),
```

Comment:
*nit (non-blocking)*: Have we implemented `fill_null`? If so, you could use that. (this is fine...)

Or introduce another column to the fixture dataframe.

Reply:
I think I like this better - makes us less dependent on what's in the input DataFrame.

fill_null is not implemented, let me add that to the drop_null issue.